api/ofdpa.proto

```diff
@@ -1,7 +1,5 @@
 syntax = "proto3";
-
 package ofdpa;
-import "stp/openconfig-spanning-tree.proto";
```

Comment:
would it be possible to extend/patch the already existing proto instead of dropping it and reinventing the wheel?

Reply:
The problem with the data model was that the data structure for PerVlan STP (PVST) was very large, and we only needed two or three variables from it. This turns out to be enough for the STP calls as well (only port name and state needed)

The Openconfig files had a PVST variable, containing list of VLANs, which again contain vlan id in one variable, and another list of interfaces again.